homestar-runtime/config/settings.toml

```diff
@@ -2,3 +2,19 @@
 process_collector_interval = 10
 
 [node]
+
+[node.network] 
```

Comment:
I say we don't need this, as defaults are merged in anyway. For testing, we can just load settings and set the keypair_config or load a fixtures config with just the change (rest of defaults will make their way through anyway, as per config.rs).

Reply:
There is an issue with the defaults merging atm since `load_settings` will fail without _all_ fields specified

Reply:
I'll double-check on this and fix if so.

Reply:
Think it was missing the default, that you added :).

Reply:
issue was with serde not filling in the defaults.

Reply:
> issue was with serde not filling in the defaults.

Yep. Just missed this.

Reply:
Of note @mriise, my next PR will clean this up for settings across the board, now with tests :).